tests/commands_test.py

```diff
@@ -43,6 +43,7 @@ async def test_commit_command(
     assert subprocess.check_output(["git", "status", "-s"], text=True) == ""
 
 
+# TODO: test without git
```

Comment:
these test changes coming in another PR?

Reply:
Yup! The Session and SessionContext still depend on git_root, so we can't use the non-git testbed in this PR

Reply:
Yup! The Session and SessionContext still depend on git_root, so we can't use the non-git testbed in this PR